e2e/ui/global-teardown.js

```diff
@@ -0,0 +1,18 @@
+import { client } from './api-client.js';
+
+async function globalTeardown() {
+  try {
+      await client(`/namespace/prod`, {method: 'delete'});
+      await client(`/namespace/dev`, {method: 'delete'});
+      await client(`/acl/policy/operator`, {method: 'delete'});
+      await client(`/acl/policy/dev`, {method: 'delete'});
+      await client(`/acl/policy/anon`, {method: 'delete'});
```

Comment:
Something I missed on the other PR: you've got a policy named `anon`. This won't get treated by Nomad the policy `anonymous`. That's implied by your needing to have a token with that policy; if it were really an anonymous policy you wouldn't need a token for it.

That being said, we already have an anonymous policy on the cluster from [`bootstrap-nomad.sh#L22-L25`](https://github.com/hashicorp/nomad/blob/main/e2e/terraform/scripts/bootstrap-nomad.sh#L22-L25) and we probably shouldn't touch it in these tests.

Reply:
Ahhh! Good call out.

Reply:
Removed it from these tests.